web/template/admin/admin_tabs/server-notifications.gohtml

```diff
@@ -14,15 +11,18 @@
 {{- /*gotype: TUM-Live/model.ServerNotification*/ -}}
 {{define "server-notification-edit"}}
     <form class="text-3 grid grid-cols-4 mb-2 p-2"
-          action="/api/serverNotification/{{if .Model.ID}}{{.Model.ID}}{{else}}create{{end}}" method="post">
+          action="/api/serverNotification/{{if .Model.ID}}{{.Model.ID}}{{else}}create{{end}}"
+          method="post"
+          x-init="new admin.ServerNotifications();">
```

Comment:
is this initialized twice on purpose?

Reply:
I think I put it there to make sure we have it if we only include `server-notification-edit` somewhere, but that doesn't really make sense here. I'll remove the second initialization. :+1: